StandaloneMmPkg/Drivers/StandaloneMmIplPei/MmFoundationHob.c

```diff
@@ -1032,3 +1032,44 @@ CreateMmFoundationHobList (
   *FoundationHobSize = UsedSize;
   return Status;
 }
+
+/**
+
+  Builds a Handoff Information Table HOB
+
+  @param MemoryBegin     - Start Memory Address.
+  @param MemoryLength    - Length of Memory.
+
+  @return EFI_SUCCESS Always success to initialize HOB.
+
+**/
+EFI_STATUS
+CreateMmHobHandoffInfoTable (
+  IN EFI_PHYSICAL_ADDRESS  MemoryBegin,
+  IN UINT64                MemoryLength
+  )
+{

```

Comment:
The function name is confusing. It creates not only HAND_OFF hob but also the END hob.

Reply:
Update the function name to CreateMmHobHandoffInfoTableAndMmHobEnd